### ISSUE TITLE
Feature: `useRender(..)`

### ISSUE THREAD

**Issue by getify** [getify/TNG-Hooks#19]
This hook is basically exactly like `useEffect(..)` except that it specifically defers its behavior **asynchronously** until the render frame (via `requestAnimationFrame(..)`).

**Comment by phenax** [comment]
Can I work on implementing this?

**Comment by getify** [comment]
Yes but I would like to hold off on for just a little bit longer. I want to land the reorg discussed in #20 first. I hope I can tackle this in the next week or two. :)

You can start on it if you want but just know the merge diffs may be numerous. :)

**Comment by phenax** [comment]
Alright
